civiform/civiform
# Localize required fields note on application summary view and make the * consistently red

### Description

Localized the required fields explanation text ("Note: Fields marked with a * are required.") on the application summary page.

Also, made the asterisk red in the explanation text of the application block view.  This means the * will be consistently red across the application.  

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)


#### User visible changes


### Instructions for manual testing

1. From the landing page, select a program and click "Apply".
2. Select a language other than English from the language dropdown.
3. Note that the required fields note is translated.
4. Click "Continue".
5. Note that the * is red.

![image](https://github.com/civiform/civiform/assets/134556723/bb090c20-78f6-421d-85f6-7ca6654271a1)


### Issue(s) this completes

Fixes #5536
